job-server/src/main/resources/shiro.ini

```diff
@@ -0,0 +1,12 @@
+
+# use this for basic ldap authorization, without group checking
+# activeDirectoryRealm = org.apache.shiro.realm.ldap.JndiLdapRealm
+# use this for checking group membership of users based on the 'member' attribute of the groups:
+activeDirectoryRealm = spark.jobserver.auth.LdapGroupRealm
+activeDirectoryRealm.contextFactory.environment[java.naming.security.credentials] = password
+activeDirectoryRealm.contextFactory.url = ldap://xxx.yyy..org:389

```

Comment:
@velvia  Yes, makes a lot of sense. I moved the file. What else needs to be done / documented to make "this part of the deply process - Docker container"?